src/validation.cpp

```diff
@@ -4142,14 +4146,16 @@ bool CVerifyDB::VerifyDB(
                 skipped_l3_checks = true;
             }
         }
-        if (ShutdownRequested()) return true;
+        if (ShutdownRequested()) return VerifyDBResult::SUCCESS;
```

Comment:
In commit "validation: Change return value of VerifyDB to enum type" (422127b696f390e7cd34e897e7d74f09e3dae2cb)

In the interest of preventing future bugs where shutdown could be interpreted as success, it seems like it would be less misleading (and easy) to return a `VerifyDBResult::INTERRUPTED` error code here, analogous to `ChainstateLoadStatus::INTERRUPTED`.

Reply:
Good point! I added a commit to do this and adjusted the release notes (since this also changes the behavior of the `-verifydb` RPC)